### ISSUE TITLE
Driver shows 0 total tasks in ClassifierDL training

### ISSUE THREAD

**Issue by willjanningCAS** [JohnSnowLabs/spark-nlp#14004]
### Is there an existing issue for this?

- [X] I have searched the existing issues and did not find a match.

### Who can help?

@maziyarpanahi 

### What are you working on?

Training a ClassifierDLApproach annotator on Bert large cased English sentence embeddings of a custom text document dataset. Training is taking place in EMR on a YARN-managed cluster.

### Current Behavior

![executors_tab](https://github.com/JohnSnowLabs/spark-nlp/assets/145142710/f624c45a-8f9b-40b0-8b0f-bfeb030d4f43)
^^^History server after app has successfully completed^^^

I am running a training app. Currently, the Spark history server shows the driver did not run any tasks over the course of training.

### Expected Behavior

I would expect the training of the ClassifierDLApproach annotator to take place entirely inside the driver. I expect this based on feedback I got from issue #13987. In other words, I'd expect the driver to have a non-zero number of total tasks.

### Steps To Reproduce

train_df = spark.read.option('header', 'true').parquet()

document_assembler = DocumentAssembler()
.setInputCol("fileText")
.setOutputCol("document")

sentenceDetector = SentenceDetectorDLModel.pretrained()
.setInputCols("document")
.setOutputCol("sentence")

bert = BertSentenceEmbeddings.pretrained("sent_bert_large_cased", "en")
.setInputCols("sentence")
.setOutputCol("bert_sentence_embeddings")

classifier_approach = ClassifierDLApproach()
.setInputCols("bert_sentence_embeddings")
.setOutputCol("class")
.setLabelColumn("article_or_other")
.setMaxEpochs(MAX_EPOCHS)
.setEnableOutputLogs(True)
.setOutputLogsPath("logs")
.setDropout(DROPOUT)

pipeline = Pipeline(
stages=[
document_assembler,
sentenceDetector,
bert,
classifier_approach])

pipeline.fit(train_df)

### Spark NLP version and Apache Spark

sparknlp version 4.0.0
spark version 3.2.1
pyspark version 3.1.2

### Type of Spark Application

spark-submit

### Java Version

1.8.0_382

### Java Home Directory

/etc/alternatives/jre

### Setup and installation

conda in EMR bootstrap script

### Operating System and Version

Amazon Linux release 2 (Karoo)

### Additional Information

This is a job training ClassifierDLApproach on Bert large sentence embeddings. For ~20 kilobytes of text it takes roughly 20 minutes to train.

**Comment by maziyarpanahi** [comment]
ClassifierDLApproach like other trainable DL-based annotators uses TensorFlow, so it's not a Spark job/task to be visible inside the Spark UI. (it has its own logs that can be seen in the console or written on disks)

**Comment by maziyarpanahi** [comment]
> closed it by mistake.

**Comment by willjanningCAS** [comment]
thank you for the response. That makes sense
